src/expressions/visitors/InvalidNode.cpp

```diff
@@ -0,0 +1,10 @@
+#include <antares/expressions/visitors/InvalidNode.h>
+
+namespace Antares::Expressions::Visitors
+{
+
+InvalidNode::InvalidNode(const std::string& node_name):
+    std::invalid_argument("Antares::Expressions::Nodes Visitor: invalid node type " + node_name)
```

Comment:
```c++
"Antares::Expressions::Nodes Visitor: ..."
```
Does user cares about hard coded namespace things when she reads the error message of an invalid node ?
```c++
"Invalid node type " + node_name
```
is enough

Reply:
I made it more "human": 
~~~
"Node visitor encountered an invalid node type: " + node_name
~~~